boards/STM32F407VET6-STM32-F4VE-V2.0.md

```diff
@@ -134,6 +134,10 @@ title: "STM32F407VET6 - STM32 F4VE V2.0"
     </tr>
 </table>
 
+## Resources
+
+ - [Schematic]({{ site.url }}/assets/pdf/boards/original-schematic-STM32F407VET6-STM32_F4VE_V2.0.pdf)
```

Comment:
Thanks for adding the schematics. You uploaded two files, but link to only one here, is there a difference?

Reply:
The other file is for issue #14 - the 407Z-board

Reply:
Oh, right. Would you mind splitting these changes off into a separate branch and pull request? Let's try to keep these two stories nicely split. Having two things change under one issue/PR is a little bit confusing, hence my original question.

Reply:
I agree that it gets confusing. I am rather new at github, so this is a learning experience. Will try to split off so there is only one board involved in each fork...

Reply:
There is now a branch for issue #14 and the files uploaded for it has been deleted from (my) "master" branch. 
The added schematics for the STM32F407VET6-STM32_F4VE_2.0 is still in the (my)"master" branch: the uploaded ...pdf file and one line added in the board file. I think leaving that might actually be less confusing than splitting and removing, but you're the boss so I will if you say so. (But haven't yet.)

Reply:
I don't exactly know what you mean with leaving files here, but as far as I can see, this diff looks good!